general-concepts/dependencies/text.xml

```diff
@@ -543,36 +584,6 @@ presence or absence.
 </subsection>
 </section>
 
-<section>
-<title>Legacy Inverse USE-Conditional Dependency Syntax</title>
-<body>
-
-<p>
-	Once upon a time the <c>:</c> conditional operator was allowed in <c>*DEPEND</c>:
-</p>
-
-<codesample lang="ebuild">
-DEPEND="use-flag? ( app-misc/foo ) : ( app-misc/bar )"
-</codesample>
-
-<p>
-<b>This syntax is no longer permitted</b>. It is exactly equivalent to the
-following, which should be used instead:
-</p>
-
-<codesample lang="ebuild">
-DEPEND="use-flag?  ( app-misc/foo )
-	!use-flag? ( app-misc/bar )"
-</codesample>
-
-<p>
-It is useful to recognise the legacy syntax and to know that it is no longer
-valid.
-</p>
-
-</body>
-</section>
-
```

Comment:
Another candidate to move to above mentioned appendix.

Reply:
Not sure if it's worth keeping in devmanual. We already have it in PMS.

Reply:
Agreed with @mgorny. It's already in the PMS.

Reply:
I have actually dropped this section with EAPI cleanup in #145.